pyRevitPlus.tab/Image.panel/Crop Image.pushbutton/script.py

```diff
@@ -105,9 +105,9 @@ def crop_image(img_path, rectangle_crop):
     return new_img_path
 
 if not __shiftclick__:
-  #__window__.Close()
-
-img_element, element_bbox = None, None
+    #__window__.Close()
+    
+    img_element, element_bbox = None, None

```

Comment:
This line should no be indented.
We can remove the shift click conditional and unindent it

Reply:
Yep that works.